terraform/etl/53-aws-glue-housing-extract-mmh-vulnerabilities-to-refined.tf

```diff
@@ -0,0 +1,67 @@
+locals {
+  housing_mtfh_case_notes_enriched_to_refined_environment_count = local.is_live_environment && !local.is_production_environment ? 1 : 0
```

Comment:
This is only for deploying to staging. If you want to your glue job deploy to Prod as well, use below
`housing_mtfh_case_notes_enriched_to_refined_environment_count = local.is_live_environment ? 1 : 0`